### DISCUSSION TITLE
Apollo Save Tool v1.4.4

### DISCUSSION THREAD

**Discussion by bucanero** [bucanero/apollo-ps4#160] [Announcements]
### Added

* 11.00 firmware support (thanks to @LightningMods @SocraticBliss)
* New cheat codes
  - Like a Dragon: Ishin!
  - Unicorn Overlord
* Custom save decryption support
  - Like a Dragon: Ishin!
  - Metal Gear Solid 5: Ground Zeroes
  - The Last of Us: Part II
  - Uncharted: Drake's Fortune Remastered
  - Uncharted 2: Among Thieves Remastered
  - Uncharted 3: Drake's Deception Remastered
  - Uncharted 4: A Thief's End
  - Uncharted: The Lost Legacy
* Custom checksum support
  - Dead Rising
  - Like a Dragon: Ishin!
  - Strider
  - The Last of Us: Part II
  - Uncharted: Drake's Fortune Remastered
  - Uncharted 2: Among Thieves Remastered
  - Uncharted 3: Drake's Deception Remastered
  - Uncharted 4: A Thief's End
  - Uncharted: The Lost Legacy

### Fixes

* Patch Code fixes
  - Metal Gear Solid 5: The Phantom Pain

### Misc

* Updated [Apollo Patch Engine](https://github.com/bucanero/apollo-lib) to v1.0.0
  - Add `jenkins_oaat`, `lookup3_little2` hash functions
  - Add `camellia_ecb` encryption
  - Add RGG Studio decryption (PS4)
  - Add Dead Rising checksum
  - Fix SaveWizard Code Type D issue with `CRLF` line breaks
  - Changed BSD command syntax for `decompress(offset, wbits)` and `compress(offset)`

<hr /><em>This discussion was created from the release <a href='https://github.com/bucanero/apollo-ps4/releases/tag/v1.4.4'>Apollo Save Tool v1.4.4</a>.</em>

**Comment by danilaltd** [comment]
Still no ps5 support? Is it in plans?

**Comment by XPLOIT-CORPSE** [comment]
Ac odyssey 1.56 data save becomes corrupted once used with the tool, even if just resigning it without enabling any of the cheats

I can't get any ss rn but once I can I will upload them

**Comment by ViniciusBCarvalho** [comment]
Hey man, first of all... Thank you very much for your work! Secondly, I would like to ask you if it's possible to add the functionality to "Disable PSN account"? I have activated my account and now it starts to download updates when I open the game. I want to disable it but the tool does nothing when I press X on top of it (it shows a closed lock on it). Thank you very much!
